tests/test_util.py

```diff
@@ -402,25 +421,38 @@ def request_callback(request):
 
                 digest = 'v1-converted-digest'
 
-        headers = {
-            'Content-Type': '{}+jsonish'.format(media_type_prefix),
-            'Docker-Content-Digest': digest
-        }
+        headers = {}
+        if digest_is_v1 or has_content_type_header:
```

Comment:
Yes, Pulp is not sending schema v1 digest in the output as this digest is cannot be used for pulling later on - https://github.com/pulp/pulp_docker/blob/master/plugins/pulp_docker/plugins/registry.py#L384. A correct manifest would be returned however, so its possible to fetch mediaType from the manifest even if the header is missing.